xonsh/vox.py

```diff
@@ -0,0 +1,206 @@
+from os.path import join, basename, exists
+import os
+from shutil import rmtree
+import venv
+
+import builtins
+
+
+class Vox:
+    """Vox is a virtual environment manager for xonsh."""
+
+    def __init__(self):
+        """Ensure that $VIRTUALENV_HOME is defined and declare the available vox commands"""
+
+        if not builtins.__xonsh_env__.get('VIRTUALENV_HOME'):
+            if os.name == 'nt':
+                home_path = builtins.__xonsh_env__['USERPROFILE']
+
+            elif os.name == 'posix':
+                home_path = os.path.expanduser('~')
```

Comment:
I'm sure this was my bad explanation in the early morning :)

On Sat, Feb 13, 2016 at 1:58 PM Konstantin Molchanov <
notifications@github.com> wrote:

> In xonsh/vox.py
> <https://github.com/scopatz/xonsh/pull/675#discussion_r52831995>:
>
> > +
> > +import builtins
> > +
> > +
> > +class Vox:
> > +    """Vox is a virtual environment manager for xonsh."""
> > +
> > +    def __init__(self):
> > +        """Ensure that $VIRTUALENV_HOME is defined and declare the available vox commands"""
> > +
> > +        if not builtins.__xonsh_env__.get('VIRTUALENV_HOME'):
> > +            if os.name == 'nt':
> > +                home_path = builtins.__xonsh_env__['USERPROFILE']
> > +
> > +            elif os.name
>
>  == 'posix':
> > +                home_path = os.path.expanduser('~')
>
> I feel so stupid right now :-)
>
> moigagoo@17cb4cb
> <https://github.com/moigagoo/xonsh/commit/17cb4cbd1708c0f45ce860240cd795f477bcde41>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scopatz/xonsh/pull/675/files#r52831995>.
>
-- 

Asst. Prof. Anthony Scopatz
Nuclear Engineering Program
Mechanical Engineering Dept.
University of South Carolina
Affiliated Professor of the University of Haifa
scopatz@cec.sc.edu
Office: (803) 777-7629
Cell: (512) 827-8239
Check my calendar
<https://www.google.com/calendar/embed?src=scopatz%40gmail.com>